### ISSUE TITLE
Add `masked_lm_labels` argument to `TFAlbertForMaskedLM`

### ISSUE THREAD

**Issue by jarednielsen** [huggingface/transformers#2848]
# 🚀 Feature request

The PyTorch `AlbertForMaskedLM` model has support for the `masked_lm_labels` parameter, while `TFAlbertForMaskedLM` does not. I'd like to bring feature parity.

It looks like a similar feature is also missing for `TFBertForMaskedLM`, `TFRobertaForMaskedLM`, `TFDistilBertForMaskedLM`. I'd be happy to add support for those models as well.

## Motivation

I'm pretraining TF NLP models, and this would simplify the training script by encapsulating the loss function.

## Your contribution

I'm happy to contribute the code. I'll follow CONTRIBUTING.md, any gotchas I should be aware of?

**Comment by mar-muel** [comment]
Hi! This feature would be great to have.

I'm curious how `TFBertMaskedLM` (and the like) are supposed to be used with the keras `fit()` functionality?

It seems like one is supposed to loop through the training data and calculate the cross-entropy loss for each batch (#2926). I see there was related discussion also here #1999.

Happy for any input!

**Comment by jarednielsen** [comment]
Closing because this feature was added in #4530
